.rubocop.yml

```diff
@@ -0,0 +1,102 @@
+inherit_gem:
+  dlss_cops: "config/dlss_baseline.yml"
+
+# Turn on RSpec cops
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.2
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
```

Comment:
`db`, I'd argue certainly. Migrations are the only user-maintained thing in there, and most of them are from upstream.

`config`, I could do either way. :shrug: